### DISCUSSION TITLE
proxy provider 支持写入到 proxyies 而不是 proxy group

### DISCUSSION THREAD

**Discussion by wlrsx** [MetaCubeX/mihomo#1428] [Q&A]
我在 proxy provider 远程文件中配置了代理，但是在clash配置文件中硬编码了Rule中的Proxy name。
如果是 proxy group 使用 use 命令就可以引用代理集合，可是我并不想配置代理组，这会提示规则找不到对应的proxy name

**Comment by xiandu999-yu** [comment]
？
点对点？那还用个鬼的clash啊...

**Reply by wlrsx** [reply]
> ？ 点对点？那还用个鬼的clash啊...

proxy provider导入到组对我没有用处，不需要组，一个规则只对应一个Proxy，只是图方便只更新 Proxy provider 就好了，嘻嘻。

**Reply by xiandu999-yu** [reply]
> > ？ 点对点？那还用个鬼的clash啊...
> 
> proxy provider导入到组对我没有用处，不需要组，一个规则只对应一个Proxy，只是图方便只更新 Proxy provider 就好了，嘻嘻。

自动选择不香吗...

**Reply by wlrsx** [reply]
> > > ？ 点对点？那还用个鬼的clash啊...
> > 
> > 
> > proxy provider导入到组对我没有用处，不需要组，一个规则只对应一个Proxy，只是图方便只更新 Proxy provider 就好了，嘻嘻。
> 
> 自动选择不香吗...

我是做运营的🤣，不同地区的站只对应一个proxy。

**Reply by xiandu999-yu** [reply]
> > > > ？ 点对点？那还用个鬼的clash啊...
> > > 
> > > 
> > > proxy provider导入到组对我没有用处，不需要组，一个规则只对应一个Proxy，只是图方便只更新 Proxy provider 就好了，嘻嘻。
> > 
> > 
> > 自动选择不香吗...
> 
> 我是做运营的🤣，不同地区的站只对应一个proxy。

6

**Reply by xiandu999-yu** [reply]
> > > > ？ 点对点？那还用个鬼的clash啊...
> > > 
> > > 
> > > proxy provider导入到组对我没有用处，不需要组，一个规则只对应一个Proxy，只是图方便只更新 Proxy provider 就好了，嘻嘻。
> > 
> > 
> > 自动选择不香吗...
> 
> 我是做运营的🤣，不同地区的站只对应一个proxy。

![image](https://github.com/user-attachments/assets/594fe164-cd31-4195-91d5-2e1c23990c2f)
配置完点点进行了[doge]

**Reply by wlrsx** [reply]
我的配置不需要proxy-groups，只需要proxy-providers引入proxies。没有办法，现在只能在策略组使用 use 引用 provider：
proxy-groups:
- name: "Hongkong_HASH"
  type: load-balance
  strategy: consistent-hashing
  <<: *g
  use:
  - Hongkong

**Comment by wlrsx** [comment]
**不需要 proxy-groups，因为我是在linux中执行mihomo的，如果只维护proxies.yaml文件就省事很多。**
proxy-providers:
  proxies:
    <<: *p
    path: ./proxy_providers/proxies.yaml
    filter: "hk|HK|hongkong|hong kong|港|HGC|HKBN|PCCW|HKT"
rules:
  - DOMAIN-SUFFIX,ipinfo.io,香港01
  - MATCH,REJECT
